app/controllers/api/v1/application_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Api::V1::ApplicationController < ActionController::API
+  respond_to :json
+end
```

Comment:
- [ ] Great job so far, 👏🏻  everything looks fine, however, I could not create a database to run the migration, I asked to my colleague to clone this and run it on his computer to check if the problem came from my side, however, he also got the same error. 🤔 Probably the problem should come from your database creation. please fix this. here is the error screenshot bellow 👇🏻 


![image](https://user-images.githubusercontent.com/79058364/217988384-16ab1298-af56-4960-a565-51a4c9259cb4.png)